app/api/server/v1/roles.js

```diff
@@ -13,6 +13,19 @@ API.v1.addRoute('roles.list', { authRequired: true }, {
 	},
 });
 
+// api to to get updated roles after a date(in ISODate format)
+API.v1.addRoute('roles.listByUpdatedDate', { authRequired: true }, {
+	get() {
+		check(this.bodyParams, {
+			updatedAfter: String,
+		});
+
+		const roles = Roles.find({ _updatedAt: { $gte: new Date(this.bodyParams.updatedAfter) } }).fetch();
```

Comment:
Can you create a Model method and put this logic inside it to avoid model logic leak?

Reply:
Yes sure. I'll do it :)